### DISCUSSION TITLE
[V3] @Stacks Blade Not Working

### DISCUSSION THREAD

**Discussion by blissjaspis** [livewire/livewire#6169] [1. Help]
Hi, I'm just try to upgrade my livewire from v2 to v3, but on the way there is a problem, my stacks blade directive not working.
In v2 it works but when upgrading to v3 it doesn't work.

Is there any documentation write about injecting script manually like stack and push blade directive?

So for now i hold my for upgrading my source code.

**Comment by samharvey44** [comment]
This is a known issue and a failing test has been submitted so hopefully it will be fixed soon.

**Comment by blissjaspis** [comment]
Oh ok, thank you for the information.
